cypress/support/commands.ts

```diff
@@ -563,3 +567,22 @@ Cypress.Commands.add("checkFlags", name => {
       return data[0].enabled;
     });
 });
+
+Cypress.Commands.add("testData", (dataToTest, index?) => {
```

Comment:
Might be a good idea to specify TS types where you can - in this command and in the rest of the commands.ts file. 
I am also getting a couple of errors in this same file (I think they are legacy errors) e.g. line 213. Happy to have a call to look through them together.